packages/react-ui-map/src/Mapbox/index.js

```diff
@@ -0,0 +1,56 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+import mapboxgl from 'mapbox-gl/dist/mapbox-gl'
+
+export default class Mapbox extends PureComponent {
+  static propTypes = {
+    size: PropTypes.string,
+    color: PropTypes.string,
+    theme: PropTypes.object,
+    token: PropTypes.string,
+    center: PropTypes.array,
+    style: PropTypes.string,
+    zoom: PropTypes.number,
+    container: PropTypes.string,
+  }
+
+  static defaultProps = {
+    theme: {},
+  }
+
+  componentDidMount() {
+    mapboxgl.accessToken = this.props.token
+    const map = new mapboxgl.Map({
```

Comment:
eventually the map will be accessible via context https://github.com/rentpath/ag.js/blob/master/src/apps/shared/Map/components/Map.js#L46 but yeah the const is not reused. 

i'll clean this up next iteration